services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/SingleRateSummarySection.tsx

```diff
@@ -91,10 +119,9 @@ export const SingleRateSummarySection = ({
     isSubmitted: boolean
     statePrograms: Program[]
 }): React.ReactElement | null => {
-    const formData: RateFormData =
-        rate?.status === 'UNLOCKED'
-            ? rate?.revisions[1].formData
-            : rate?.revisions[0].formData
+    const rateRevision =
+        rate?.status === 'UNLOCKED' ? rate?.revisions[1] : rate?.revisions[0]
```

Comment:
This was causing a bug - forgot that draft revisions don't come down on the rate revisions anymore. This should always be the first revision in the list.